back-end/src/lms/models/students.py

```diff
@@ -18,15 +18,28 @@
 )
 
 
+class StudentPost(models.Model):
+    title = models.CharField(primary_key=True,
```

Comment:
`primary_key=True` is bad. @gakhromov

Reply:
Let's leave it as is for now. We will return to this issue later